File: leetcode/832. Flipping an Image/solution.cpp
#include <bits/stdc++.h>

using namespace std;

class Solution {
public:
    vector<vector<int>> flipAndInvertImage(vector<vector<int>>& A) {
        vector<vector<int>> res;
        for (int i = 0; i < A.size(); i++) {
            vector<int> tmp;
            for (int j = A[i].size() - 1; j >= 0; j--) tmp.push_back(1 - A[i][j]);
            res.push_back(tmp);
        }
        return res;
    }
};

File: leetcode/832. Flipping an Image/solution.py
class Solution:
    def flipAndInvertImage(self, A: List[List[int]]) -> List[List[int]]:
        invert = lambda x: 1 if x == 0 else 0
        return [list(map(invert, A[i][::-1])) for i in range(len(A))]
